.golangci.yml

```diff
@@ -75,3 +78,4 @@ linters:
 
     # Mostly just annoying
     - exhaustruct
+    - funlen
```

Comment:
I think the funlen is annoying especially implement test

Reply:
I agree, but only for test. So please move this exclude rule under https://github.com/yitsushi/go-misskey/blob/main/.golangci.yml#L30